src/gui/forcefieldTab.cpp

```diff
@@ -479,7 +539,36 @@ void ForcefieldTab::on_MasterTermAddBondButton_clicked(bool checked)
     dissolveWindow_->setModified();
 }
 
-void ForcefieldTab::on_MasterTermRemoveBondButton_clicked(bool checked) { Messenger::error("NOT IMPLEMENTED YET.\n"); }
+void ForcefieldTab::on_MasterTermRemoveBondButton_clicked(bool checked)
+{
+    QMessageBox queryBox;
+    queryBox.setWindowTitle(QString("Remove Master Bond"));
+    queryBox.setText(QString("This operation cannot be undone!"));
```

Comment:
Worth elaborating here a bit and elsewhere (and reformatting because this will no doubt break the CI!):
```suggestion
    queryBox.setText(QString("Are you sure? This operation cannot be undone!\nANy uses of this term in species will be converted to a local definition."));
```